sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/command.py

```diff
@@ -272,7 +284,7 @@ def code(self, value: str) -> None:
             msg = "Can't set code property for a registered component {}"
             raise ValidationException(
                 message=msg.format(self.component),
-                no_personal_data_message=msg,
+                no_personal_data_message=msg.format(self.component),
```

Comment:
Is this intentional ? Can you please try rebasing from main ?

Reply:
reverted this change